### PR TITLE
start_all shell file 

### PR DESCRIPTION
**Pull request by Jsyro**
one shell file starts the von-network, did resolver, and 2 BPA's. 

register_did's wasn't properly replaceing the .env key.

### CODE REVIEW

File: docker/bbp-network/start_all.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```
Comment (parc-jason) [inline docker/bbp-network/start_all.sh:1]: very cool.
 wonder if you can incorporate register-dids in here, maybe with an env var or param to call it or not

**Review by usingtechnology — COMMENTED** [review]
cool.
looks like there may be some duplicated/redundant lines?

File: docker/bbp-network/start_all.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+#GET LOCAL LEDGER GENESIS FILES AND START RESOLVER
+curl --retry-connrefused --retry 5 --retry-delay 2 http://localhost:9000/genesis -o resolver/localhost_9000.txn
+
+while [ $? -ne 0 ];
+do
+    sleep 3
+    curl --retry-connrefused --retry 5 --retry-delay 2 http://localhost:9000/genesis -o resolver/localhost_9000.txn
+done
+docker-compose -f ./resolver/docker-compose.yml up -d driver-did-sov
```
[the remark this answers is not in the document]
Comment (Jsyro) [reply docker/bbp-network/start_all.sh:16]: yea.. let me look into that. that does look wierd.

File: docker/bbp-network/start_all.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+#DL, BUILD, RUN, LOCAL HYPERLEDGER
+git clone https://github.com/bcgov/von-network.git von-network
+./von-network/manage build
+./von-network/manage start
+
+#GET LOCAL LEDGER GENESIS FILES AND START RESOLVER
+curl --retry-connrefused --retry 5 --retry-delay 2 http://localhost:9000/genesis -o resolver/localhost_9000.txn
```
[the remark this answers is not in the document]
Comment (Jsyro) [reply docker/bbp-network/start_all.sh:9]: if this tries to call before the von-network is ready. it get's `connection reset by peer` and it doesn't retry. It appears there is no curl option that will retry on that error type. so i had to code in the loop myself.

[the remark this answers is not in the document]
Comment (parc-jason) [reply docker/bbp-network/start_all.sh:9]: the loop isn't the problem, the first call is necessary if you go into the loop?  why not just enter the loop?

**Review by usingtechnology — DISMISSED** [review]

**Review by usingtechnology — APPROVED** [review]
👍

**Pull request merged**